pkg/localcontroller/managers/lifelonglearning/lifelonglearningjob.go

```diff
@@ -281,6 +281,10 @@ func (lm *Manager) evalTask(job *Job) error {
 				jobConfig.UniqueIdentifier, err)
 		}
 
+		// 这个触发器是指，LC已经告诉GM可以触发Eval worker了，
```

Comment:
please comment by english

Reply:
The comment has been changed to English.